ci/docs/pulp_index_pulp3.rst

```diff
@@ -42,4 +41,4 @@ Index and Search
 .. toctree::
    :maxdepth: 1
 
-   all_content_index_pulp3
+   all_content_index
```

Comment:
@goosemania oh I gotcha. 
I reviewed this because parts of it should be merged with you pulp plugin docs pr right?

I think this is good to merge as is if it blocks getting that merged. We can always change this code later